lib/jxl/dec_ans.h

```diff
@@ -275,25 +275,36 @@ class ANSSymbolReader {
     return static_cast<uint32_t>(ret);
   }
 
-  // Takes a *clustered* idx. Can only use if HuffRleOnly() is true.
-  void ReadHybridUintClusteredHuffRleOnly(size_t ctx,
-                                          BitReader* JXL_RESTRICT br,
-                                          uint32_t* value, uint32_t* run) {
+  // Takes a *clustered* idx, implicit ctx==1. Can only use if HuffRleOnly() is
+  // true.
+  void JXL_INLINE ReadHybridUintClusteredHuffRleOnly(BitReader* JXL_RESTRICT br,
```

Comment:
Probably also should mention it's 000/100 only...

Reply:
At this point one might replace "HuffRleOnly" with something like FJXLFastPath...

Reply:
Makes sense, done.